src/muya/lib/index.js

```diff
@@ -372,6 +378,60 @@ class Muya {
     return this.keyboard.hideAllFloatTools()
   }
 
+  /**
+   * Replace the word range with the given replacement.
+   *
+   * @param {*} line A line block reference of the line that contains the word to
+   *                 replace - must be a valid reference!
+   * @param {*} wordCursor The range of the word to replace (line: "abc >foo< abc"
+   *                       whereas `>`/`<` is start and end of `wordCursor`). This
+   *                       range is replaced by `replacement`.
+   * @param {string} replacement The replacement.
+   * @param {boolean} setCursor Shoud we update the editor cursor?
+   */
+  replaceWordInline (line, wordCursor, replacement, setCursor = false) {
```

Comment:
I prefer put this method on `ContentState.prototype`, for example create a file name `spellCheckerCtrl.js`.

```js
ContentState.prototype. replaceWordInline = function () {}
```

Reply:
That's actually a good idea but for a high level API or Muya API to transform lines. Do you have any ideas where to add the API interface?

Reply:
The API interface need to put on the muya instance, 

```js
Muya {
  replaceWordInline (...args) {return this.contentState. replaceWordInline(...args)}
}
```

is it ok?

Reply:
How about to add a `core` `contentState` class/file with all basic editor operations like replace, add line, remove line, etc?

Reply:
>How about to add a core contentState class/file with all basic editor operations like replace, add line, remove line, etc? So we could easily integrate the operation with the interface.

Yeh, we need to abstract some operations, and then add them when optimizing muya.